docs/src/pages/customization/globals/globals.md

```diff
@@ -1,38 +1,66 @@
 # Globals
 
-<p class="description">The styleOverrides key enables you to customize the appearance of all instances of a component type, while the props key enables you to change the default value(s) of a component's props.</p>
+<p class="description">With the `theme.components` key you can customize the appearance of all instances of a component; change the default value(s) of a component's props; and add custom variants to your components.</p>
 
-## CSS
-
-When the configuration variables aren't powerful enough, you can take advantage of the
-`styleOverrides` key of the `theme` to potentially change **every single style** injected by Material-UI into the DOM.
-That's a really powerful feature.
+## Default props
 
-To override lab components styles with TypeScript, check [this documentation](/components/about-the-lab/#typescript).
+You can change the default props of every Material-UI component.
+A `defaultProps` key is exposed in the theme's `components` key for this use case.
 
 ```js
 const theme = createMuiTheme({
   components: {
-    // Style sheet name ⚛️
-    MuiButton: {
-      styleOverrides: {
-        // Name of the rule
-        textPrimary: {
-          // Some CSS
-          color: 'white',
-        },
+    // Name of the component ⚛️
+    MuiButtonBase: {
+      defaultProps: {
+        // The props to change the default for.
+        disableRipple: true, // No more ripple!
       },
     },
   },
 });
 ```
 
-{{"demo": "pages/customization/globals/GlobalCss.js"}}
+{{"demo": "pages/customization/globals/DefaultProps.js"}}
+
+To override lab component styles with TypeScript, check [this page](/components/about-the-lab/#typescript).
 
-The list of these customization points for each component is documented under the **Component API** section.
-For instance, you can have a look at the [Button](/api/button/#css).
+## Theme variables
 
-## Global CSS
+Another way to override the look of all component instances is to adjust the [theme configuration variables](/customization/theming/#theme-configuration-variables).
+
+```jsx
+const theme = createMuiTheme({
+  typography: {
+    button: {
+      fontSize: '1rem',
+    },
+  },
+});
+```
+
+{{"demo": "pages/customization/globals/ThemeVariables.js"}}
+
+## Global CSS override
+
+Components expose [global class names](/styles/advanced/#with-material-ui-core) to enable customization with CSS.
+This is very similar to how you would customize Bootstrap.
```

Comment:
Assumes familiarity with bootstrap. Is that a given any more?

Reply:
Removed